Arithmetic.pp

```diff
@@ -49,6 +49,7 @@
 %token  minus     \-|−
 %token  times     \*|×
 %token  div       /|÷
+%token  indices   \^
```

Comment:
I think using either `indices` or `exponent` would be more correct but it's a trivial thing, I don't mind either way.